core/templates/pages/donate-page/donate-page.component.spec.ts

```diff
@@ -69,7 +68,7 @@ describe('Donate page', () => {
   let translateService: TranslateService;
   let pageTitleService: PageTitleService;
   let windowRef: MockWindowRef;
-  let mailingListBackendApiService: MailingListBackendApiService;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
These changes are really required.?

Reply:
Oh, the `alertsService` is currently not in use and therefore I am removing it from the test file. Thanks